ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -51,7 +51,7 @@ module.exports =  Marionette.ItemView.extend({
         },
         handleTrash: function() {
             let loginUser = user.get('user');
-            if(loginUser.get('email') === this.model.get('createdBy'))
+            if(loginUser.get('email') !== this.model.get('createdBy'))
```

Comment:
❓ I might just be reading this wrong, but it would seem like this would allow anyone _other than_ the user who created the form to be able to attempt to delete it. Wouldn't we really want the decision to be made by the backend where it can (in theory) take into account more information in the policy decision like user roles and such? (i.e. users in admin group supersede normal users, etc)

Reply:
Do we want to display the trashcan to people who can't delete it though? We can make the decision on the back end, but it should manifest here.

Reply:
@mackncheesiest is correct and the backend should now validate the subject during the deletion phase in this PR https://github.com/codice/ddf/pull/3200/files#diff-0cff128da7b84e6e61b5370f3c0a5ec5R121

As far as visibility goes, that's a good point, but might confuse the user if some have a trashcan and some don't. Wdu think?